### PR TITLE
Set log level to debug if app is not released

### PR DESCRIPTION
**Pull request by holzeis**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -133,8 +133,13 @@ class _TenTenOneState extends State<TenTenOneApp> {
   }
 
   Future<void> _callSync() async {
-    final balance = await api.getBalance();
-    if (mounted) setState(() => balanceModel.update(Amount(balance.confirmed)));
+    try {
+      final balance = await api.getBalance();
+      balanceModel.update(Amount(balance.confirmed));
+      FLog.trace(text: 'Successfully synced wallet');
```
Comment (klochowicz) [inline lib/main.dart:139]: is there a way of enabling these logs? like using some sort of config file for Flutter or CLI arguments?

**Review by klochowicz — APPROVED** [review]
looks good, thanks :)

**Review by da-kami — APPROVED** [review]

File: lib/main.dart (hunk 1 above)
Comment (holzeis) [reply lib/main.dart:139]: hmm.. we could use --dart-define https://itnext.io/flutter-1-17-no-more-flavors-no-more-ios-schemas-command-argument-that-solves-everything-8b145ed4285d

**Pull request merged**